File: Models/Route.cs
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

using GridWebApp.Models.Interfaces;

namespace GridWebApp.Models
{
    [Table("routes")]
    public class Route : IModel, IData
    {
        [JsonProperty("ID")]
        [Key]
        public long Id { get; set; }

        [JsonProperty("PARENT_REGION_ID")]
        public int ParentId { get; set; }

        [JsonProperty("REGION_NAME")]
        public string Name { get; set; }


        [JsonProperty("VALUE_WOMEN_KID")]
        public int WomenKid { get; set; }

        [JsonProperty("VALUE_WOMEN_ADULT")]
        public int WomenAdult { get; set; }

        [JsonProperty("VALUE_WOMEN_SENIOR")]
        public int WomenSenior { get; set; }


        [JsonProperty("VALUE_MEN_KID")]
        public int MenKid { get; set; }

        [JsonProperty("VALUE_MEN_ADULT")]
        public int MenAdult { get; set; }

        [JsonProperty("VALUE_MEN_SENIOR")]
        public int MenSenior { get; set; }


        [JsonProperty("VALUE_SUM")]
        public int Sum { get; set; }


        [JsonProperty("CITY_FROM_ID")]
        public long CityFromId { get; set; }

        [JsonProperty("CITY_TO_ID")]
        public long CityToId { get; set; }


        [JsonProperty("CITY_FROM")]
        public string CityFrom { get; set; }

        [JsonProperty("CITY_TO")]
        public string CityTo { get; set; }

        public void CalcSum()
        {
            Sum = MenKid + MenAdult + MenSenior + WomenKid + WomenAdult + WomenSenior;
        }
    }
}

File: ClientApp/src/app/_models/route.ts
export class Route {
    id: number;
    parent_id: number;
    name: string;
    women_kid: number;
    women_adult: number;
    women_senior: number;
    men_kid: number;
    men_adult: number;
    men_senior: number;
    sum: number;
    city_from_id: number;
    city_to_id: number;
    city_from: string;
    city_to: string;

    constructor(data: any) {
        this.id = data.id;
        this.parent_id = data.parentId;
        this.name = data.name;
        this.women_kid = data.womenKid;
        this.women_adult = data.womenAdult;
        this.women_senior = data.womenSenior;
        this.men_kid = data.menKid;
        this.men_adult = data.menAdult;
        this.men_senior = data.menSenior;
        this.sum = data.sum;
        this.city_from_id = data.cityFromId;
        this.city_to_id = data.cityToId;
        this.city_from = data.cityFrom;
        this.city_to = data.cityTo;
    }
}
